powsybl-parent/pom.xml

```diff
@@ -35,14 +35,35 @@
         <maven.buildnumber.version>1.4</maven.buildnumber.version>
         <maven.checkstyle.version>3.1.0</maven.checkstyle.version>
         <maven.failsafe.version>2.22.2</maven.failsafe.version>
-        <maven.gmavenplus.version>1.6.1</maven.gmavenplus.version>
+        <maven.gmavenplus.version>1.8.1</maven.gmavenplus.version>
+        <!--
+            For groovydoc, we must use the same version as the project's groovy version. Standardize on the groovy.version property.
+            For projects which don't use groovy at all, this means the pluginmanagement will hold an invalid <version> definition
+            (the literal string '${groovy.version}' instead of a valid version like '2.5.8') but this doesn't break the build.
+            For projects which do use groovy (ie at least one submodule declares gmavenplus in their <build><plugins> element)
+            then the build will break until either groovy.version or maven.groovydoc.version is set. It will break not just for the release profile
+            so projects can detect errors early and fix their groovydoc configuration.
+        -->
+        <maven.groovydoc.version>${groovy.version}</maven.groovydoc.version>
```

Comment:
Should we define a default value for this property?

Reply:
I don't think so.
With a default value, you get an error only when you do ```$ mvn package -Prelease```.
Without a default value, you get an error when you do ```$ mvn package```.

I think the second behavior is better, it avoids discovering errors when you are about to release